src/language-handlebars/printer-glimmer.js

```diff
@@ -438,9 +518,12 @@ function printOpenBlock(path, print) {
   );
 }
 
-function printElseBlock(node) {
+function printElseBlock(node, options) {
+  const escape =
+    options.htmlWhitespaceSensitivity !== "strict" ? [hardline] : [];
+
   return concat([
-    hardline,
+    ...escape,
```

Comment:
```suggestion
  return concat([
    options.htmlWhitespaceSensitivity !== "strict" ? hardline : "",
```

No need use `array`

Reply:
it is a trick to avoid pushing an empty string in the concat args ^^

Reply:
but if you prefer, I will remove the array

Reply:
I fixed this :)